CPS-????/README.md

```diff
@@ -0,0 +1,81 @@
+---
+CPS: ?
```

Comment:
```suggestion
CPS: 4
```

Reply:
Committing so we can become more ambitious about cross-referencing this with other CIPs e.g. https://github.com/cardano-foundation/CIPs/pull/466